### ISSUE TITLE
Bug when I am downloading a Big Playlist

### ISSUE THREAD

**Issue by Neko-no-akuma-TW** [ErrorFlynn/ytdlp-interface#95]
There are some trouble when I am downloading the big youtube playlist. Playlist has 185 items, when the ytdlp-interface is downloading 117th items, there is no error notifiction, the downloading program exited. I have no idea why this bug happened, so I put that at here.

**Comment by ErrorFlynn** [comment]
Can you post the link to the playlist?

**Comment by Neko-no-akuma-TW** [comment]
> Can you post the link to the playlist?

https://youtube.com/playlist?list=PLxihdffDFJ4QtjV2A-pfKLkanv-09vdn_
Here it is. By the way, there is only 76 items, it can download.

**Comment by ErrorFlynn** [comment]
I was not able to reproduce the issue.

**Comment by Neko-no-akuma-TW** [comment]
> I was not able to reproduce the issue.
Uhh. Fine, I will try to redownload the software, maybe it will be fine

**Comment by ErrorFlynn** [comment]
If you continue having this issue, you could copy the command line that the program uses, and run it in a Command Prompt window.

First start the download (so that the command line appears in the output view), then stop the download, right-click on the output, and select `Copy command only`. Then open a command prompt (`Win+R`, type `cmd` and press Enter), paste the command line and press Enter. Then let the download finish, or press `Ctrl+C` to stop it if you change your mind.

**Comment by ErrorFlynn** [comment]
I have now experienced a crash when downloading a Bandcamp playlist (album), but I was unable to reproduce it with the debug build of the program. I will continue to try to resolve this, but it will probably take a while, especially since I have little time or energy to work on the project right now.

**Comment by Neko-no-akuma-TW** [comment]
I see. By the way, I still can't download after I redownloaded the software.

**Comment by Neko-no-akuma-TW** [comment]
"D:\應用程式\VideoDownloadSoftware\yt-dlp.exe" --no-mtime  -P C:\Users\Keyne\Downloads -o "%(playlist_title)s\%(playlist_index)d - %(title)s.%(ext)s" "https://www.youtube.com/playlist?list=PLxihdffDFJ4QtjV2A-pfKLkanv-09vdn_"

Here is the download command. Maybe it can help you to find the bug

**Comment by Neko-no-akuma-TW** [comment]
Oh, I use the command for download the videos, there is no trouble, so I think that is ytdlp interface bug

**Comment by ErrorFlynn** [comment]
I experienced a repeatable crash using your command line, but it happened inside the GUI library, not in the application code. The cause of the crash doesn't make any sense to me (a memory deallocation error that shouldn't be happening). Maybe it's some kind of unlikely toolchain issue, or maybe I'm missing something.

Either way, I've modified the line in the GUI library that was causing the crash, as a workaround. That seems to have eliminated the crash for me, but I'm not 100% sure that my crash is the same as what you or others experience, so I'll leave this issue open for now. In any case, this change will be in of the next version.
